### ISSUE TITLE
[Findability] User can't determine next steps in a flow.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#78134]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** OCTO Health Apps
**Product:** MHV Medical Records
**Feature:** After-Visit Summary

## Findings details
**VA.gov Experience Standard - issue:** User can't determine next steps in a flow.
**VA.gov Experience Standard - category:** Findability
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Under "Your health information as of this appointment," the "Problem list," "Immunizations," "Allergies and adverse drug reactions," "Recent lab results," "My medications," and "My VA supplies" sections can potentially be long and don't have much structure to them to help screen reader users navigate. 

Screen reader users often rely on structural elements to help them get a sense for how much content is in a section or what they can skim through or skip to. When you have a long section of content with no structure that could be used for this kind of wayfinding, screen reader users may either have to listen to everything --- which is frustrating, if they're just looking for one specific piece of information.

## Recommended action
To the extent that it's possible with the data you're receiving, it would be great to add a little extra structure. If there's a way to add headings or list semantics to this content, that would be huge for improving navigability for screen reader users. 

Example: Could each allergy/drug reaction (currently separated by an `hr`) be presented as a `ul`/`li` list? Even if it's styled to look identical to what you have now, screen reader users would hear "list with X items" when arriving at that block of content, and screen reader users would be able to more easily skip from item to item. Likewise for the other sections.

I realize that you have some limitations based on what you're getting, but a lot of the page prior to this point has these structural elements that can assist in navigation. The list of upcoming appointments is semantically presented as a list *and* has headings for each appointment, there are several lists present under the "Your treatment plan" section, and so on. For a screen reader user, it's like the structure of the page just kind of drops off a cliff three quarters of the way down. While most of the page is relatively easy to navigate, you run into this big chunk that's semantically just a giant wall of text.

It looks like each chunk of content in these problem sections is structured in a relatively consistent way: a `div` with a `p` inside of it containing the text. Headings would be great where possible, but hopefully inserting list semantics would be relatively straightforward.

## References

- **WCAG Success Criteria:** [WCAG 2.4.6 Headings and Labels - Level AA](https://www.w3.org/TR/WCAG21/#headings-and-labels)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
